canonical/tdx
# PEK 764 cleanup and get snap working

* Move pytests to tests directory
* Get snap/checkbox to properly find the installed snap tests with
  checkbox
* Move pytests to use tox instead of our custom run.sh script
* Add pytest categories to call via "tox -e <category>"
* Add checkbox categories to call via
  "checkbox-tdx-classic.test-runner-automated-<category>"
* Get rid of redundant TDXTEST_IMAGE_FILE environment variable and use
  existing TDXTEST_GUEST_IMG (and don't copy it)
* Use relative checkbox-cli path to be compatible to using other runtime
* In pytest, check for image specification and existence
* Make sure TDXTEST_GUEST_IMG variable is passed to tox
  environments (e.g. checkbox22, checkbox24, ...)
